### DISCUSSION TITLE
［求助］我的绳网处在等上和等不上的叠加态了

### DISCUSSION THREAD

**Discussion by Huanshenxuanlong** [share121/inter-knot#2472] [其他]
不知道为什么就是看不到帖子，但好像能发帖，而且github是登录状态的
![Screenshot_2024-08-17-00-38-55-95_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/f0e4c10f-a060-4f04-9bcc-eec1a0c6f8aa)

**Comment by cpu644** [comment]
你好！这里是绳网技术支援团队。请检查你的绳网跨域助手是否为最新版（截至现在1.14.0）。

**Reply by Huanshenxuanlong** [reply]
重新装了一下，好了

**Reply by ysd1911** [reply]
咋重装的，我现在也是这样，叠加态，加载要好久

**Reply by Huanshenxuanlong** [reply]
坏了，重装也不行了，加速器是好的，就是绳网的界面登不进去，帖子加载不出来

**Reply by ysd1911** [reply]
挂了梯子时好时坏的，只能从GitHub上看

**Comment by FurinaShark** [comment]
电脑的话可以试试CTRL+SHIFT+R刷新，我就这样弄好了

**Reply by Yulooooo** [reply]
这个委托我接了

**Comment by qmtqyh** [comment]
刚起床的时候想上绳网看看结果也是楼主那样，最后发现是没挂梯子TvT

**Comment by iuhjk** [comment]
我也不知道为啥，手机上也是这样，电脑上到是好好的

**Comment by missfggc** [comment]
我是清除一下数据在重装就好了
